.npmignore

```diff
@@ -0,0 +1,7 @@
+.github/**
+CODE_OF_CONDUCT.md
+SECURITY.md
+SUPPORT.md
+tsconfig.json
+.eslintrc.js
+api.ts
```

Comment:
Might as well make this a `*.ts` exclusion instead of just `api.ts`.

Reply:
Changing.